gradle.properties

```diff
@@ -20,11 +20,14 @@ cascadingVersion = 2.6.3
 # Spark
 spark13Version = 1.6.2
 spark20Version = 2.3.0
+spark30Version = 3.0.0
```

Comment:
Would be nice to add support to hdp 3.2 as well as spark 3 supports now hadoop 3 too.

Reply:
added new distro option `hadoopYarn3` which builds using hadoop 3.2